UnityAuth/src/main/java/io/unityfoundation/auth/AuthController.java

```diff
@@ -41,31 +41,31 @@ public HttpResponse<HasPermissionResponse> hasPermission(@Body HasPermissionRequ
 
     Optional<Tenant> tenantOptional = tenantRepo.findByName(requestDTO.tenantId());
     if (tenantOptional.isEmpty()) {
-      return createHasPermissionResponse(false, "Cannot find tenant!");
+      return createHasPermissionResponse(false, null,"Cannot find tenant!");
```

Comment:
I think it is safe to include `user.getEmail()` for all responses.  We are not revealing any information since it was provided as an input.